README.md

```diff
@@ -1,12 +1,116 @@
-# Template Playground
+<h1 align="center">Accord Project Template Playground</h1>
+<p align="center">
+  <a href="https://github.com/accordproject/models/blob/master/LICENSE"><img src="https://img.shields.io/github/license/accordproject/models" alt="GitHub license"></a>
+  <a href="https://www.accordproject.org/">
+    <img src="https://img.shields.io/badge/powered%20by-accord%20project-19C6C8.svg" alt="Accord Project" />
+  </a>
+</p>
 
-A web playground for Accord Project templates:
+This repository hosts the Accord Project Playground. Templates are resourced using the [Template-Engine](https://github.com/accordproject/template-engine) Rich-text templates are defined in TemplateMark (either as markdown files, or JSON documents) and are then merged with JSON data to produce output documents.
+
+The build system for this repository publishes the templates to: [https://playground.accordproject.org](https://playground.accordproject.org)
+
+## Accord Project Playground
+
+**Project Overview**
+
+The Accord Project Playground is a web-based platform designed to empower developers of all backgrounds to learn and experiment with Accord Project functionalities in a user-friendly and interactive environment. It is a Web playground for Accord Project templates:
 
 1. TemplateMark for the natural language text
-2. Concerto for the data model
+2. [Concerto](https://docs.accordproject.org/docs/model-concerto.html) for the data model
 3. TypeScript for logic within the templates
 4. Markdown-transform to transform the output to HTML, PDF etc.
```

Comment:
```suggestion
4. Markdown-Transform to transform the output to HTML, PDF etc.
```
May be you can even add links to TemplateMark, Markdown-Transform and the Template-Engine repos here.

Reply:
Sure, Sure.
Noted, Resolved ✅